test/generate-vdm.spec.ts

```diff
@@ -0,0 +1,152 @@
+/*!
+ * Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+
+const warn = jest.fn(message => console.log('MOCKED WARNING: ', message));
+jest.mock('@oclif/command', () => {
```

Comment:
You can also remove this `warn` part as this mock is not needed for the generator as far as I can tell.